apps/entry/serializers.py

```diff
@@ -437,6 +439,11 @@ def add_comment_text(self, comment, text):
         )
 
     def validate(self, data):
+        assignees = data.get('assignees')
+        entry = data.get('entry')
+        is_assignee = set(ProjectMembership.objects.filter(project=entry.project, member__in=assignees).values_list('member', flat=True).distinct()) == set([a.id for a in assignees])
+        if not is_assignee:
+            raise serializers.ValidationError({'assignees': "Cannot assign assignees to this entry_comment"})
```

Comment:
@AdityaKhatri Is this error message enough for the client-side?

Reply:
@k9845 , lets change this to:
`Selected assignees don't belong to this project`.